client/src/components/TextArea/TextArea.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./TextArea.css";
+
+const TextArea = props => (
+  <div>
+    <label className={props.titleClassList}>{props.title}</label>
```

Comment:
These classes should be part of the component, not passed in as a prop

Reply:
Also, someone may or may not pass in a label. Can you add a conditional here that makes the label optional?